mountinfo/mounted_linux_test.go

```diff
@@ -15,252 +15,247 @@ import (
 
 // tMount is a wrapper for unix.Mount which is used to prepare test cases.
 // It skips the test case if mounting is not possible (i.e. user is not root),
-// adds more context to mount error, if any, and installs a cleanup handler to
-// undo the mount.
-func tMount(t *testing.T, src, dst, fstype string, flags uintptr, options string) error {
+// fails the test case if there is a mount error, and installs a cleanup
+// handler to undo the mount.
+func tMount(t *testing.T, src, dst, fstype string, flags uintptr) {
 	if os.Getuid() != 0 {
 		t.Skip("root required for mounting")
 	}
 
-	err := unix.Mount(src, dst, fstype, flags, options)
+	err := unix.Mount(src, dst, fstype, flags, "")
 	if err != nil {
-		return &os.PathError{Path: dst, Op: "mount", Err: err}
+		t.Helper()
+		t.Fatal(&os.PathError{Path: dst, Op: "mount", Err: err})
 	}
 	t.Cleanup(func() {
 		if err := unix.Unmount(dst, unix.MNT_DETACH); err != nil {
+			t.Helper()
 			t.Errorf("cleanup: unmount %q failed: %v", dst, err)
 		}
 	})
-	return nil
+}
+
+func tCreateFile(t *testing.T, dir string) string {
```

Comment:
This should probably be a `t.Helper()` as well?

Reply:
Right.